spicy/toolchain/src/ast/types/unit-items/field.cc

```diff
@@ -37,12 +37,12 @@ static Type _adaptType(const type::unit::item::Field& field, const Type& t, bool
 
 Type spicy::type::unit::item::Field::parseType() const {
     if ( isContainer() ) {
-        Type etype = _originalType().as<type::Vector>().elementType();
+        Type etype = originalType().as<type::Vector>().elementType();
```

Comment:
While it looks like we cannot change `originalType` to return a `const T&` instead of a value (AFAICT it is ineligible for const lifetime extension), let's at least cache its return value here so we don't have to call it twice.

Reply:
Ack